resources/public/api/conf/2.0/application.raml

```diff
@@ -789,7 +789,7 @@ types:
                   /annual-returns:
                     post:
                       is: [headers.acceptHeader, headers.contentHeader]
-                      displayName: "[Sandbox only] Send an annual return of information"
+                      displayName: "Send an annual return of information"
                       description: "##This API is currently not available in Production.
```

Comment:
If [Sandbox only] has been removed, is the description `##This API is currently not available in Production.` still valid?
Did this change form part of the ticket?

Reply:
It was a change that was meant to be done a while ago that never happened. The Production available isn't applicable anymore. I'll change that.

Reply:
Also have updated the data in testdata 2.0. The postman collection looks fine to us as this call isn't concerning the superseded withdrawal and runs fine when the collection is run.